articles/storage-actions/storage-tasks/storage-task-quickstart-portal.md

```diff
@@ -24,6 +24,8 @@ In this quickstart, you learn how to use the [Azure portal](https://portal.azure
 - An Azure subscription. See [create an account for free](https://azure.microsoft.com/free/?WT.mc_id=A261C142F).
 
 - An Azure storage account. See [create a storage account](../../storage/common/storage-account-create.md). As you create the account, make sure to enable version-level immutability support and that you don't enable the hierarchical namespace feature.
+  
+- -During the public preview, you can target only storage accounts that are in the same region as the storage tasks.
```

Comment:
```suggestion
   During the public preview, you can target only storage accounts that are in the same region as the storage tasks.
```

Reply:
Looks like you want to amplify this requirement beneath the storage account prerequisite. I assume some support cases have come in when folks have attempted to complete this QuickStart? Anyway, I would remove the bullet as this requirement relates to the storage account bullet. Can you pls accept the suggestion? Thx!

Reply:
Thank you, I moved ahead and committed your suggestion. Request you to merge if it's good to go.